doc/getting_started/beginner_tutorial.rst

```diff
@@ -25,163 +22,245 @@ Create a Repo
+b90afeb1-c12b-4ca5-a4f4-50c50efb20bb   edges/da51395708cb4812bc8695bb151b69e3   2 minutes ago      1 seconds            success
+9182d65e-ea36-4b98-bb07-ebf40fefcce5   edges/4dd2459531414d80936814b13b1a3442   5 minutes ago      3 seconds            success
+ ...
 
-One thing that's interesting to note is that our pipeline is completely incremental. Since ``grep`` is a ``map`` operation, Pachyderm will only ``grep`` the new data from set2.txt instead of re-filtering all the data. If you look back at the "sum" pipeline, you'll notice the ``method`` and that our code uses ``/pfs/prev`` to compute the sum incrementally based upon our previous commit. You can learn more about incrementally in our advanced :doc:`../advanced/incrementality` docs.
+.. code-block:: shell
 
-We can view the parental structure of the commits we just created.
+# View the output data
 
-.. code-block:: shell
+# on OSX
+$ pachctl get-file edges master AT-AT.png | open -f -a /Applications/Preview.app
+
+$ pachctl get-file edges master kitten.png | open -f -a /Applications/Preview.app
+
+# on Linux
+ $ pachctl get-file edges master AT-AT.png | display
 
- $ pachctl list-commit data
- BRANCH              REPO/ID             PARENT              STARTED             FINISHED            SIZE
- master              data/master/0       <none>              19 minutes ago      19 minutes ago      874 B
- master              data/master/1       master/0            2 minutes ago       2 minutes ago       874 B
+ $ pachctl get-file edges master kitten.png | display
+ ...
+
+We can view the parental structure of the commits we just created.
 
+.. code-block:: shell
 
-Exploring the File System
+# List the inputs commits
+$ pachctl list-commit images
+REPO                ID                                 PARENT                             STARTED            DURATION             SIZE
+images              d86dfa815a26419db2dd0f55ae9bed8a   b22e3d89461f4161abf8d671919c2597   2 minutes ago      Less than a second   238.3 KiB
+images              b22e3d89461f4161abf8d671919c2597   634aa3728141442f8537a7b1bb6aefbf   2 minutes ago      Less than a second   136 KiB
+images              634aa3728141442f8537a7b1bb6aefbf   <none>                             13 minutes ago     Less than a second   57.27 KiB
+
+# List the output commits
+$ pachctl list-commit edges
+REPO                ID                                 PARENT                             STARTED            DURATION             SIZE
+edges               8848e11056c04518a8d128b6939d9985   da51395708cb4812bc8695bb151b69e3   2 minutes ago      Less than a second   133.6 KiB
+edges               da51395708cb4812bc8695bb151b69e3   4dd2459531414d80936814b13b1a3442   2 minutes ago      Less than a second   59.38 KiB
+edges               4dd2459531414d80936814b13b1a3442   <none>                             5 minutes ago      Less than a second   22.22 KiB
+
+Exploring the File System (optional)
```

Comment:
This is kind of cool, but I just wonder whether it's worth including here or maybe just including in the cookbook or somewhere.  There is the danger of them opening the directory structure and getting confused about how things are organized, duplicated, etc.  Not sure.  I trust your judgement either way.  Just provided an alternative point of view.